src/components/Form.js

```diff
@@ -5,7 +5,7 @@ import Button from './Button'
 import Input from './Input'
 import { CrossIcon } from '../lib/Icons'
 
-function Form({ userPlace, setUserPlace, status }) {
+function Form({ status, startSearch }) {
     const [userInput, setUserInput] = useState('')
     const [validation, setValidation] = useState(true)
```

Comment:
Instead of setting the initial state of  ``validation`` to ``true``, my first intention would be to set it to ``false``, as it cannot be true at the beginning / at the first key down, can it?

Reply:
I'am using that state to show a message if the input isn't valid. The validation checks at the submit event. So validation is true until someone submits something invalid, then i'ts false and the message is shown. As long as there is a valid input submitted.